### ISSUE TITLE
//paste no longer functioning

### ISSUE THREAD

**Issue by StealWonders** [IntellectualSites/FastAsyncWorldEdit#614]
# Bug report for FastAsyncWorldEdit 1.16.3

**FastAsyncWorldEdit Version Number:** Bukkit-Official(1.16-356;0685881)
**Spigot/Paper Version Number:** git-Paper-195 (MC: 1.16.3)

**Description of the problem:**
Using the `//paste` command no longer pastes anything that I have copied to my clipboard.

**How to replicate**:
Copy and paste something.

**Checklist**:
- [ ] I included all information required in the sections above
- [x] I made sure there are no duplicates of this report
- [x] I made sure I am using an up-to-date version of 
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by lusu007** [comment]
Same overhere. I can provide some more information if needed.

FAWE-Version: 357
Paper: git-Paper-196

Debugpaste:
https://athion.net/ISPaster/paste/view/257cdb0477054a91ad053943c031402a

Plugins:
Arceon, BuildersUtilities, BungeeTabListPlus, Essentials, FastAsyncWorldEdit, goBrush, goPaint, HeadDatabase*, Hyperverse, LegendEssentials, LuckPerms, Vault, VoidGenerator, VoxelSniper, WorldEdit, WorldGuard

**Comment by Victorsueca** [comment]
I managed to stumble upon this issue on my server after updating it from 1.16.2 to 1.16.3 and updating FAWE to 1.16-357.

Here's my debugpaste:
https://athion.net/ISPaster/paste/view/5313e8f7880c4756869e2ecd86e91c34

Furthermore, I found a weird structure that appeared at the border of my load distance, which seems to be related to my several failed attempts to paste.

![image](https://user-images.githubusercontent.com/11365204/93098836-9780e680-f6a7-11ea-8dfb-3038cb32f6a5.png)
(Backgroud: Original stone and grass cuboid that I was attempting to copypaste. Front: Weird structure that appeared at the border of the loaded chunks.)
